README.adoc

```diff
@@ -213,18 +232,20 @@ query/SystemResource.java
 include::finish/query/src/main/java/io/openliberty/guides/query/SystemResource.java[]
 ----
 
-Inject the [hotspot=systemPropertiesProperty file=1]`system.properties` configuration property as a `List<String>` type.
+Inject the `system.properties` property by using the [hotspot=systemPropertiesInject file=1]`@Inject` and [hotspot=systemPropertiesProperty file=1]`@ConfigProperty` annotations with a variable type of [hotspot=listStringType file=1]`List<String>`.
 
-Then, modify the [hotspot=getSystemPropertiesMethod file=1]`getSystemProperties()` method. Iterate through the [hotspot=systemProperties file=1]`systemProperties` list to retrieve all the specified property values from the `systemClient` REST client.
+The [hotspot=listStringType file=1]`systemProperties` variable retrieves the list of system properties from the configuration and passes it to the [hotspot=getSystemPropertiesMethod file=1]`getSystemProperties()` method.
 
-Check out the service that you modified at the http://localhost:9085/query/systems/localhost URL. You can see the specified system properties in JSON format.
+The [hotspot=getSystemPropertiesMethod file=1]`getSystemProperties()` then iterates through this list of system properties by using the [hotspot=systemProperties file=1]`systemProperties` variable to retrieve its value from the `system` service.
+
+Point your browser to the http://localhost:9085/query/systems/localhost URL. You can see the list of system properties that are retrieved from the `system.properties` property in JSON format.
 
 // ===========================================================================================
 == Using configuration source APIs
 
-MicroProfile Config provides https://openliberty.io/docs/latest/reference/javadoc/microprofile-5.0-javadoc.html?package=org/eclipse/microprofile/config/package-frame.html&class=org/eclipse/microprofile/config/package-summary.html[configuration source APIs^] for retrieving details about configuration properties. You can use the `ConfigValue` API class to retrieve various useful information about a single specified configuration property. Alternatively, you can use the `Config` API class to look up the metadata of all underlying configuration properties. 
+MicroProfile Config provides https://openliberty.io/docs/latest/reference/javadoc/microprofile-5.0-javadoc.html?package=org/eclipse/microprofile/config/package-frame.html&class=org/eclipse/microprofile/config/package-summary.html[configuration source APIs^] for obtaining detail information about your application's configuration. These APIs can be used to retrieve various useful information about a single specified configuration property by using the `ConfigValue` API class, or to lookup the metadata of all underlying configurations of your microservice by using the `Config` API class.
```

Comment:
same here- whats the benefit?

Reply:
Edited in https://github.com/OpenLiberty/draft-guide-microprofile-config-apis/pull/17/commits/ae426b2edaa323381d31a623506df60e6220e304